fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTray.kt

```diff
@@ -6,27 +6,114 @@ package org.mozilla.fenix.tabstray
 
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.google.accompanist.pager.ExperimentalPagerApi
+import com.google.accompanist.pager.HorizontalPager
+import com.google.accompanist.pager.rememberPagerState
+import kotlinx.coroutines.launch
+import org.mozilla.fenix.compose.Divider
 import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.compose.button.PrimaryButton
 import org.mozilla.fenix.theme.FirefoxTheme
 
 /**
  * Top-level UI for displaying the Tabs Tray feature.
  */
+@OptIn(ExperimentalPagerApi::class)
+@Suppress("LongMethod")
 @Composable
 fun TabsTray() {
-    Box(
-        modifier = Modifier.fillMaxSize()
+    val pagerState = rememberPagerState(initialPage = 0)
+    val scope = rememberCoroutineScope()
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
             .background(FirefoxTheme.colors.layer1),
     ) {
-        Text(
-            text = "Tabs Tray to Compose",
-            color = FirefoxTheme.colors.textPrimary,
-            style = FirefoxTheme.typography.body1,
-        )
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(all = 16.dp),
+        ) {
+            Text(
+                text = "Temporary tab switchers",
+                color = FirefoxTheme.colors.textPrimary,
+                style = FirefoxTheme.typography.body1,
+            )
+
+            Spacer(modifier = Modifier.height(16.dp))
+
+            PrimaryButton(text = "Normal tabs") {
+                scope.launch {
+                    pagerState.animateScrollToPage(Page.NormalTabs.ordinal)
+                }
+            }
+
+            Spacer(modifier = Modifier.height(16.dp))
+
+            PrimaryButton(text = "Private tabs") {
+                scope.launch {
+                    pagerState.animateScrollToPage(Page.PrivateTabs.ordinal)
+                }
+            }
+
+            Spacer(modifier = Modifier.height(16.dp))
+
+            PrimaryButton(text = "Synced tabs") {
+                scope.launch {
+                    pagerState.animateScrollToPage(Page.SyncedTabs.ordinal)
+                }
+            }
```

Comment:
nit: It might make future refactoring easier and we might be able to remove the long method suppression if we threw this into a standalone function. The `scope.launch` repetitions could be nicely abstracted as an `onPagerClick` lambda. It's also possible doing that kind of thing this early won't be helpful so up to you

Reply:
I'm actually about 95% done with the [immediate ticket after this](https://bugzilla.mozilla.org/show_bug.cgi?id=1816743), which primarily addresses your suggestion. I'll refactor the scope.launch here though.

Reply:
Eh if it's already done in the next one I don't see a real reason to